### DISCUSSION TITLE
Sharing example of how this theme can be used as template for research projects

### DISCUSSION THREAD

**Discussion by juan11iguel** [hugo-themes/toha#1012] [Show and tell]
Hello, first of all I would like to thank all the contributors that made this theme possible.

Recently we have made public the webpage for a national research project, which can be found in [solhycool.psa.es](https://solhycool.psa.es/). The webpage uses this theme, where we took advantage of its flexibility, and managed to refocus it from a personal webpage to an  institutional/research project one.

![image](https://github.com/user-attachments/assets/32f6bb33-e910-4a2b-a1ad-fdb1b409122e)
![image](https://github.com/user-attachments/assets/e853e04a-6d4e-4c54-aa40-97d352fb5a37)

There are still some pending details to fix/improve  #992 #993 #994, and if I have time I will try to fix them myself and submit a PR, but I am very happy how it turned out, thanks again!

Best regards,
Juanmi

**Comment by hossainemruz** [comment]
Thank you for sharing.
